plugins/hydra_ax_sweeper/example/conf/config.yaml

```diff
@@ -0,0 +1,33 @@
+defaults:
+  - hydra/sweeper: ax
+
+# Banana function
+# https://en.wikipedia.org/wiki/Rosenbrock_function
+banana:
+  # To be minimized
+  x: ???
+  y: ???
+  
+
+hydra:
+  ax:
+    experiment:
+      # Banana function should be minimized (Ax defaults to maximize)
+      minimize: true
```

Comment:
I think we should default to minimize like nevergrad.
it makes more sense for ML.
Can be a in followup diff standardizing the two sweepers.

Reply:
Changed the default to min via https://github.com/facebookresearch/hydra/pull/434/commits/ceac692d0027b76f615f1b349418eafba3d62dc6